src/components/checkbox/_checkbox.scss

```diff
@@ -38,8 +38,7 @@
     width: rem(18px);
     margin-right: .5rem;
     background-color: $ui-01;
-    border-radius: 2px;
-    border: 2px solid $ui-05;
+    border: $checkbox-border-width solid $ui-05;
```

Comment:
@aagonzales, @hellobrian and I just switched it so the checkmark takes up all available space, rather than a hardcoded value.